paket.dependencies

```diff
@@ -3,6 +3,13 @@ group neutral
   framework: netstandard2.0
   source https://api.nuget.org/v3/index.json
   source https://www.myget.org/F/oxyplot
+  github fsprojects/FSharp.Compiler.PortaCode:master src/ProjectCracker.fs
```

Comment:
Now that those files evolve independently of Fabulous, maybe we need to force a specific commit rather than `master`?

Reply:
The lock file has the specific commit - if a future `paket update` causes problems that are difficult to solve then at that point we should pin in the dependencies (and then do further updates step by step). Until then tracking master on each `paket update` is ok - the Fabulous tests still test everything all over again.